tests/std/tests/GH_003570_allocate_at_least/test.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <deque>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <syncstream>
+#include <utility>
+#include <vector>
+
+using namespace std;
+
+struct {

```

Comment:
Style: We typically name structs, even in test code. (And we separate struct definitions from object definitions.)